en/flight_controller/raspberry_pi_pilotpi_ubuntu_server.md

```diff
@@ -0,0 +1,293 @@
+# PilotPi with Ubuntu Server
+
+> **Warning** Some of users may run Ubuntu Server on RPi 4B, which will consume a lot of current and generate a lot of heat. Please take more care about the safety.
```

Comment:
What are you saying here. Are you saying that you should not run on RPi 4B?  Or are you saying that you can, but if so you need to worry about safety? What aspect? Carry a fire extinguiser, or just that it won't fly well?

Reply:
How do you think about now?

Reply:
Better :-). I subedited a bit more too :-)